appengine/pond/duck/online/js/online.js

```diff
@@ -76,6 +76,9 @@ Pond.Duck.Online.init = function() {
               + '/pond-duck-online?lang='+ BlocklyGames.LANG;
           window.location = url;
         });
+    var iframe = document.getElementById('leaderboard-iframe');
+    iframe.src = 'pond-duck-leaderboard?lang='+ BlocklyGames.LANG + '&duck=' +
+        Pond.Duck.Online.duckKey;
```

Comment:
Sorry for late suggestion, but I think we could set the duckKey before we render the template and the pass it in directly. That's how it is done in Gallery.init as well.